src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -563,6 +568,73 @@ private static IEnumerable<string> DiscoverBindingTypes(IEnumerable<FunctionMeta
             return bindingTypes;
         }
 
+        // Validate that for any precompiled assembly, all functions have the same configuration precedence.
+        // Also load any extensions from the precompiled directory.
+        private void VerifyPrecompiled(Collection<FunctionMetadata> functions)
```

Comment:
nit: Could we have a better name for this method? It wasn't clear to me what `VerifyPrecompiled` meant. Perhaps `ValidateDirectLoadConfiguration`?